golem_messages/message/__init__.py

```diff
@@ -0,0 +1,146 @@
+# For backwards compatibility import all old messages by name
+# pylint: disable=unused-import
+# pylint: disable=cyclic-import
+from golem_messages.message.base import Message  # noqa
```

Comment:
maybe all those `# noqa` blocks can also be a `pylint: disable=` before the block

Reply:
if `pycodestyle` used block I would prefer block :)

Reply:
damn you `pycodestyle`! ;)